app/models/review.rb

```diff
@@ -0,0 +1,11 @@
+class Review < ActiveRecord::Base
```

Comment:
This was fixed in PR #13 (https://github.com/mcon85/doors_at_eight/pull/13). We pulled in those changes and future commits no longer get that hound notice.